README.md

```diff
@@ -343,25 +343,34 @@ A great way to contribute to Fly-Pie is creating a translation to another langua
 We suggest using a tool like [Poedit](https://poedit.net/) or the [GNOME Translation Editor](https://wiki.gnome.org/Apps/Gtranslator).
 
 Translations of Fly-Pie are stored in the `po/` directory.
-Just edit an existing translation or create a new one by opening the template [`po/flypie.pot`](po/flypie.pot) with either of these tools.
+Either edit an existing translation or create a new one by opening the template [`po/flypie.pot`](po/flypie.pot) with either of these tools.
+When editing an existing translation, please make sure that you work from the latest template by executing this command:
 
-Once you are happy with your translation, save a `<country code>.po` file in the `/po` directory and run the following command to compile the strings:
+```bash
+update-po.sh <LANG-CODE>  # <LANG-CODE>` is the language code for the translation (`de` for German, `it` for Italian etc.)
+```
+
+Once you are happy to test your translation, save it as `<LANG-CODE>.po` file in the `/po` directory and install the extension with your updated translations:
 
 ```bash
-./compile-locales.sh
+./create-release.sh -i
```

Comment:
This depends on where the Fly-Pie directory is. If the user checked out the git repository to `.local/share/gnome-shell/extensions` (as I usually do), then `./compile-locales.sh` is required. In fact it is `./scripts/compile-locales.sh` :smile:

Reply:
Forgive me, but I don't understand - why would you have to call `scripts/compile-locales.sh` when using a configuration like yours? This script gets called by `scripts/create-release.sh -i` anyway, or am I missing something here?

Reply:
Well you can try it out yourself! Just follow the clone-from-GitHub guide (https://github.com/Schneegans/Fly-Pie#c-cloning-the-latest-version-with-git), run `./create-release.sh -i` and see what happens.

In fact, the `./create_release.sh -i` is pretty harmful when used this way as it will *delete* the cloned git repository (including all un-pushed changes). I think we should add a check in this script that it does not attempt to install if it is executed from `.local/share/gnome-shell/extensions/flypie@schneegans.github.com`!

Btw, the `flypie@schneegans.github.com/compile-locales.sh` in this guide needs to be replaced with `flypie@schneegans.github.com/scripts/compile-locales.sh`, I just realized.

Does this make things clearer? If not I will try to explain it in another way :D

Reply:
Aah, now I get it. You are using the repo as installed extension, so to speak - and by passing `-i`, the zip gets unpacked over the repo itself! This is bad, yes... But easy to patch :)

I have a separate `~/Development` directory where I have the git repo, so I never ran into problems there.

Reply:
Development is a tiny bit faster when directly checked out to the extensions directory. For some changes GNOME Shell does not need to be restarted; e.g. changes to the `settings.ui` need only re-opening of the settings dialog. So I can skip the installation call in between.